assignments/arrays.js

```diff
@@ -75,22 +81,34 @@ console.log();
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+
+for(let i = 0; i < inventory.length; i++) {
+  carModels.push(inventory[i].car_model);
+}
+carModels.sort();
+
+console.log(carModels);
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for (var i = 0; i < inventory.length; i++) {
+   carYears.push(inventory[i].car_year);
+}
+carYears.sort();
+
+console.log(carYears);
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars =[];
-console.log(); 
+for (var i = 0; i < inventory.length; i++) {
+   oldCars.push(inventory[i].car_year) >== 2000;
```

Comment:
The ` >== 2000` isn't being read at all in this case. You'd want to do an if statement inside your forLoop and do something like `if (inventory[i].car_year >= 200){}` . You also only put one equal sign ` >== 2000` would be ` >= 2000`.